tests/ShadowDOM/js/Document.js

```diff
@@ -230,14 +233,18 @@ htmlSuite('Document', function() {
 
     div.offsetHeight;
 
-    var list = document.querySelectorAll('div /deep/ aa');
```

Comment:
I'd like to wait for a quick confirm from @jmesserly on the dupe tests before we remove em :)